system-uptime-tracking/xml/art_system-uptime-tracking.xml

```diff
@@ -39,6 +40,23 @@
         for analytics purposes.
       </para>
     </abstract>
+    <meta name="title" its:translate="yes">System Uptime Tracking</meta>
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+    <meta name="description" its:translate="yes">How to monitor the uptime of a client system</meta>
+    <meta name="social-descr" its:translate="yes">Monitoring the uptime of a client system</meta>
+    <meta name="task" its:translate="no">
+      <phrase>Subscription Management</phrase>
+    </meta>
+    <revhistory xml:id="rh-article-system-uptime-tracking">
+      <revision>
+        <date>???</date>
```

Comment:
@tbazant : This needs to be filled in whenever the article will be published on d.s.c (for now, it's only published as preview on sd.gh.io for our beta customers)